worker/internal/benthos/transformers/generate_street_address.go

```diff
@@ -20,16 +20,28 @@ type Address struct {
 }
 
 func init() {
-	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length"))
+	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length")).Param(bloblang.NewInt64Param("seed"))
 
 	err := bloblang.RegisterFunctionV2("generate_street_address", spec, func(args *bloblang.ParsedParams) (bloblang.Function, error) {
 		maxLength, err := args.GetInt64("max_length")
 		if err != nil {
 			return nil, err
 		}
 
+		seed, err := args.GetOptionalInt64("seed")
+		if err != nil {
+			return nil, err
+		}
+
+		var randomizer *rand.Rand
+		if seed == nil {
+			randomizer = rand.New(rand.NewSource(int64(rand.Int())))
```

Comment:
I think you can just use `rand.Int63` and have it return an int64 for you automatically.
```suggestion
			randomizer = rand.New(rand.NewSource(rand.Int63()))
```

Reply:
yup you're right!